README.md

```diff
@@ -42,9 +42,38 @@ For example:
 
 Call this to attach general tracking information to the current page. This is useful if the data is not associated with a specific event, and will be sent to the server the next time the logs are flushed.
 
-### `$logger.metric(<event>, <payload>);`
+### `$logger.metricCounter(<event>, <payload>);`
 
-Queues a metric
+Queues a counter metric, helper wrapping `logger.metric`
+
+```
+logger.metricCounter({
+  namespace: "pp.team.product.feature",
+  event: "button_click",
+  dimensions: {
+    type: "paypal"
+  }
+})
+```
+
+### `$logger.metricGauge(<event>, <payload>);`
+
+Queues a gauge metric, helper wrapping `logger.metric`
```

Comment:
Is gauge a standard term here because I think this could use a bit more detail (I can kind of intuit what it means but it's not obvious)

Reply:
it is, that's what its called in the OpenTelemetry library: https://opentelemetry.io/docs/specs/otel/metrics/api/#gauge